src/components/elements/Feedback.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components'
+import { LIGHT_RED, FONT_FAMILY } from '../../config/styles'
+import Link from '../navigation/Link'
+import { QuestionMarkIcon } from '../icons'
+
+const FeedbackWrapper = styled.div`
+  ${FONT_FAMILY};
+  border: 2px dashed ${LIGHT_RED};
+  padding: 1rem;
+  margin: 1rem 0;
+  max-width: 320px;
+  display: flex;
+  align-items: center;
+`
+const Feedback = () => (
+  <FeedbackWrapper>
+    <QuestionMarkIcon />
+    Any questions?{' '}
+    <Link to="#contact-us" style={{ marginLeft: 4 }}>
```

Comment:
@shakersinthedark it not a good practice to mix styled-components style with inline style (inline style = style={{ marginLeft: 4 }}). The reason is inline styles can't be overriden by CSS Cascade. In this case, why don't you add in the "FeedbackWrapper" something like:
```
const FeedbackWrapper = styled.div`
  a {
     margin-left:4px;
  }
   ${FONT_FAMILY};
```

Reply:
Yeah, I tried this but it didn't seem to work but I'll give it another go with added caffeine 😄

Reply:
You're a master @alexlbr. works a treat. 👍